.github/ISSUE_TEMPLATE/Ask_Question.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Ask Question
+about: Ask a question about usage or feature
+
+---
+
+### Your question
+
+describe your question clearly
+
+### Your scenes
+
+describe your use scenes (why need this feature)
+
+### Your advice
+
+describe the advice or solution you'd like
+
+### Environment
+
+- SOFARPC version:
```

Comment:
please change this to SOFA ARK, as well as Bug_Report.md

Reply:
My fault, I did not notice this... @QilongZhang Please create another PR to fix